Adding Archives documentation (#637)
SEKOIA-IO/documentation: docs/xdr/features/investigate/archives.md

## The Archives page

The Archives feature is accessible on the `Archives page` in the Operations Center menu. 

On this page, a table lists all archives created by the user. 

| Column | Description |
| --- | --- |
| Name | Name of the archive |
| Provider | Where the archive is stored |
| Number of snapshots | How many snapshots were made since the archive was first created   |
| Created at | Creation date |
| Status  | Could be `success` or `error` |

### Views from archive tab

The archives tab lists all the different restorations that have been realized on this community. This view is only accessible internally for now.

## Create an archive

To create an archive, follow these instructions:

=== "Amazon S3"

    1. **Choose your archive destination**: it can be Amazon S3 storage or OVHcloud Object storage
    2. **Configure your bucket** (the data associated with the storage space) by providing the following information
        
        | Name | Description |
        | --- | --- |
        | Container/Bucket | Name of the bucket |
        | Region | The region associated with the bucket |
        | Access Key ID | ? |
        | Access Key Secret | ? |

>     3. **Describe your archive **

Author: ```suggestion
    3. **Describe your archive**
```

    4. **Add GPK Key to access your events (optional)**: Archives are stored as GPG encrypted files so that the selected cloud provider does not have access to your events. You will always be able to load your archives inside SEKOIA.IO but if you want to access your events outside of the app, you can add your own public key.

=== "OVHCloud"

    1. **Choose your archive destination**: it can be Amazon S3 storage or OVHcloud Object storage
    2. **Configure your bucket** (the data associated with the storage space) by providing the following information
        
        | Name | Description |
        | --- | --- |
        | User name |  |
        | Password |  |
        | Tenant name |  |
        | Container name |  |

    3. **Describe your archive**
    4. **Add GPK Key to access your events (optional)**: Archives are stored as GPG encrypted files so that the selected cloud provider does not have access to your events. You will always be able to load your archives inside SEKOIA.IO but if you want to access your events outside of the app, you can add your own public key.

!!! info

>     The format used in archives are logs compressed in JSON format and it keeps enrichments made by SEKOIA.IO. Since we keep the original logs and message fields, the volumes are therefore larger in the archives compared to the originals.

Author: ```suggestion
    The format used in archives are logs compressed in JSON format with enrichments made by SEKOIA.IO. Since we keep the original logs and message fields, the volumes are therefore larger in the archives compared to the originals.
```

## How to access saved archives and re-hydrated events

When the archives feature is made available in a community, the Events page shows a new filter to list re-hydrated events. 

1. Click on `Search history` next to the search bar and select Re-hydrated events
2. Select which archive you want to upload 
3. Click on `Search` and wait for your re-hydrated logs to appear on the Events page 

!!! tip 

>     Change the date range if you cannot find events you’re looking for. 

Author: ```suggestion
    If you can't find events you're searching for, try changing the date range.
```
